test/quantization/test_quantize_fx.py

```diff
@@ -2129,31 +2137,37 @@ def forward(self, x, y):
                 ns.call_method("to"): 3 if is_scalar else 4
             }
             self.checkGraphModeFxOp(
-                OpRelu(is_inplace_op, is_functional_relu, is_scalar),
+                BinaryOpRelu(binary_op, ibinary_op, is_inplace_op, is_functional_relu, is_scalar),
                 data, quant_type, custom_qconfig_dict=custom_qconfig_dict,
-                expected_node_occurrence=node_occurrence,
-                print_debug_info=True)
+                expected_node_occurrence=node_occurrence)
 
 
     @skipIfNoFBGEMM
     def test_add(self):
```

Comment:
maybe `_test_binary_op_int8_impl` and `_test_binary_op_float16_impl` could become their own tests, and same for the others?

Reply:
hmmm, it's a trade off I think, we can split but then we'll have more tests and it also increases complexity. but i don't have a very strong preference, please let me know if you feel it's worthwhile to make this trade off.

Reply:
will land for now, we can split in a separate PR if needed.